ci-operator/config/openstack-k8s-operators/openstack-ansibleee-operator/openstack-k8s-operators-openstack-ansibleee-operator-main.yaml

```diff
@@ -8,7 +8,15 @@ build_root:
   image_stream_tag:
     name: release
     namespace: openshift
-    tag: golang-1.18
+    tag: rhel-8-release-golang-1.18-openshift-4.12
+images:
```

Comment:
FYI, we are adding a ci image, to avoid this C&P everywhere[1]
But we can change that later, once we validate that works as expected.

[1] https://github.com/openshift/release/pull/35666